tests/integration_test/action_service_test.py

```
@@ -2,6 +2,10 @@
 import datetime
 import os
 from urllib.parse import urlencode, urljoin
+from kairon.shared.utils import Utility
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff (0.8.0)

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)